src/CacheFactory.php

```diff
@@ -26,7 +27,7 @@
 use function is_string;
 
 /**
- * @method Cache __invoke(ContainerInterface $container)
+ * @method Cache|CacheItemPoolInterface __invoke(ContainerInterface $container)
```

Comment:
This seems like a BC Break - is this an `|` or an `&`?

Reply:
it is `|` but it depends on how you call this factory.
Since it is actually widening the return value it also requires the configuration of a project to change.
So it does not change anything for existing projects.